README.md

```diff
@@ -29,38 +29,52 @@ The project [data dictionary](data_dictionary.md).
 
 Clone this repository:
 
-```
+```bash
 git clone https://github.com/Mesnage-Org/PGFinder.git
 cd pgfinder
 ```
 
 Install for development:
 
-```
+```bash
 pip install -e .
+pip install -e .[tests]
 ```
 
 You're fine to use a different virtual environment, if you want!
+
 ## Usage
 
-```
+A demo is available with...
+
+```bash
 python demo.py
 ```
 
+You can also use the command line interface `find_pg.py` which works with a YAML configuration file (see
+`config/example.yaml`) for an example which you can modify. You must supply at least one option on the command line `-c
+<path/to/config.yaml>`, so to use the example config you would.
+
+``` bash
+./find_pg.py -c config/example.yaml
```

Comment:
```suggestion
./find_pg -c config/example.yaml
```

Reply:
This won't work on Windows - maybe put a note to this effect. We can address later.